EasyReflectometry/experiment/model.py

```diff
@@ -1,4 +1,4 @@
-__author__ = 'github.com/arm61'
+__author__ = "github.com/arm61"
```

Comment:
is there a GOOD reason to switch to double quotes?
- single quotes are good for quick assignments where noone expects `'` in the content
- they are easier to type (single keystroke on US/Eng keyboard) 
- reasonably standard according to highest voted answer on SO on this topic...
Does `ruff` prefer double quotes?

Reply:
The only reason why I "introduced" double quotes is because ruff defaults to this.

If preferred we can use single quotes.

Reply:
Changed back to single quotes